<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rental_Form</class>
 <widget class="QWidget" name="rental_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>도서 대출</string>
  </property>
  <widget class="QLineEdit" name="num_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="barcode_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>바코드 입력</string>
   </property>
  </widget>
  <widget class="QLabel" name="num">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>제목</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>저자</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>출판사</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>발행연도</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>총페이지</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="publisher_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="release_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="page_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rental_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>대출</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>반납</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>barcode_btn</sender>
   <signal>clicked()</signal>
   <receiver>rental_Form</receiver>
   <slot>input_barcode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rental_btn</sender>
   <signal>clicked()</signal>
   <receiver>rental_Form</receiver>
   <slot>book_rental()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>return_btn</sender>
   <signal>clicked()</signal>
   <receiver>rental_Form</receiver>
   <slot>book_return()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>input_barcode()</slot>
  <slot>book_rental()</slot>
  <slot>book_return()</slot>
 </slots>
</ui>
